bridge/src/lib.rs

```diff
@@ -101,13 +103,71 @@ methods!(
     TemporalBridge,
     _rtself, // somehow compiler is sure this is unused and insists on the "_"
 
-    fn create_connection(runtime: AnyObject, host: RString) -> AnyObject {
-        let host = host.map_err(VM::raise_ex).unwrap().to_string();
+    fn create_connection(runtime: AnyObject, options: AnyObject) -> AnyObject {
         let runtime = runtime.unwrap();
         let runtime = runtime.get_data(&*RUNTIME_WRAPPER);
 
+        let options = options.map_err(VM::raise_ex).unwrap();
```

Comment:
Is it okay to panic here? What are the implications?

Reply:
Yeah, the conversion code and error propagation can use some cleaning up.
In TS I made some extra effort to associate the property name with the conversion / validation errors.
All of this can be done later once we have a functioning SDK.